source/documentation/principles/frontend-development-principles.html.md.erb

```diff
@@ -22,15 +22,50 @@ because it covers the broad principles and is generally right. What follows are
 MOJ-specific recommendations.
 
 ## 2. Make your service accessible
-Your site must be compliant with level AA of the [Web Content Accessibility Guidelines 2.0](https://www.w3.org/WAI/intro/wcag). You should also try assistive
-technology tools as much as you can. However that’ll be far from giving you a
-complete view of the issues your service might have. With a bit of luck to will
-be assessed by proper accessibility experts (like the [Digital Accessibility Centre](https://www.digitalaccessibilitycentre.org/)). But don’t wait for it to
-happen to implement accessibility principles. Another helpful thing to do is
-automatic accessibility testing as part of your continuous integration.
-
-- [pa11y](https://github.com/springernature/pa11y)
-- Automated Accessibility Testing in Rails with capybara-accessible: `https://blogs.vmware.com/vsphere/2012/03/creating-custom-metadata-using-the-vcloud-api.html`
+
+In the UK, the [Public Sector Bodies Accessibility Regulations](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps) mean we are legally required to make sure our digital services meet level AA of the latest version of the [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG22/). The latest version of these guidelines is 2.2, which was released in October 2023.
```

Comment:
> The latest version of these guidelines is 2.2, which was released in October 2023.

I can see this being a bit of maintenance nightmare depending on how often they update 🔧 Think we should just omit it and rely on them clicking the link? Maybe re-word the text to explicitly mention the latest version can be found at [Web Content Accessibility Guidelines](https://www.w3.org/TR/WCAG22/)  🤔

Reply:
Good suggestion will amend.